lms/static/sass/discussion/views/_home.scss

```diff
@@ -0,0 +1,166 @@
+// discussion - home layout
```

Comment:
Is this partial's job just to style the default landing page for discussions or is this more general for both that view and other views/states users navigate into (discussion detail/thread)?